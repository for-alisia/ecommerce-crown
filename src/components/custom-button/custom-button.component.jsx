/** Libraries */
import React from 'react';

/** Styles */
import './custom-button.styles.scss';

export const CustomButton = ({ children, isGoogleSignIn, inverted, ...otherProps }) => (
  <button className={`${inverted ? 'inverted' : ''} ${isGoogleSignIn ? 'google-sign-in' : ''} custom-button`} {...otherProps}>
    {children}
  </button>
);
