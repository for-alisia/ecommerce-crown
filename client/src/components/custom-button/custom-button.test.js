// @ts-nocheck
/** Libraries */
import { mount } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';

/** Components */
import { CustomButton } from './custom-button.component';

describe('Custom button component', () => {
  const wrapper = mount(<CustomButton />);

  it('expects to match a snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
