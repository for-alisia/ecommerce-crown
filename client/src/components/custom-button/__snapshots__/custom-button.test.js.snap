// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom button component expects to match a snapshot 1`] = `
<CustomButton>
  <styled.button>
    <button
      className="kvfyMz"
    />
  </styled.button>
</CustomButton>
`;
